FAQ.md

```diff
@@ -0,0 +1,75 @@
+# Frequently Asked Questions
```

Comment:
I think the rendering hereis also weird.

Reply:
Can you be more precise? What's bothering you? The rendering in my browser is OK (different from in the Hedgedoc pad). 

![Screenshot from 2025-02-11 20-50-22](https://github.com/user-attachments/assets/b3ab9c42-9bbd-485f-b2c5-85de42d92c2e)

Reply:
![Screenshot_20250211-205454](https://github.com/user-attachments/assets/c797eb0b-f227-40ee-ab36-a0a4192f85dc)

This is how it looks on vanadium (chrome) on mobile. The summary line is detached from the triangle.

Reply:
Is this caused by the extra newlines around the headers between the `<summary>` tags

Reply:
yeah, or the handling of the `####` heading within the summary. is there `<bold> What is ... </bold> or similar?

Reply:
No, apparently the only way to have md rendering inside <summary> is to have these extra newlines...